package.json

```diff
@@ -1,11 +1,16 @@
 {
   "name": "breakdancer",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "A breakpoint tracking utility",
-  "main": "index.js",
+  "main": "lib/index.js",
   "scripts": {
-    "test": "mocha --compilers js:babel/register ./test.js",
-    "lint": "godaddy-js-style-lint index.js test.js"
+    "prebuild": "rimraf lib",
+    "build": "babel src -d lib",
+    "lint": "gulp lint",
+    "unit": "gulp unit-coverage",
+    "pretest": "npm run lint",
+    "test": "npm run unit",
+    "prepublish": "npm run test && npm run build"
```

Comment:
I think this will make the install fail, `prepublish` runs on install and since the `src` folder is excluded from the npm package the test cannot run